packages/gradle/src/plugin/nodes.ts

```diff
@@ -139,13 +134,17 @@ export const createNodes: CreateNodes<GradlePluginOptions> = [
       calculatedTargets[hash] = {
         name: projectName,
         targets,
-        targetGroups,
+        metadata: {
+          targetGroups,
+          technologies: ['gradle'],
```

Comment:
Remove this from the project, only have it on the targets.

Reply:
removed